cfme/web_ui/menu.py

```diff
@@ -439,16 +439,19 @@ def f(_):
 
         def f2(_):
             args = [toplevel, secondlevel, thirdlevel]
-            loc = cls.ROOT
-            for arg in args:
-                if arg:
-                    if arg.startswith("/"):
-                        loc_to_use = cls.NAMED_LEV_HREF
+            if version.current_version() == 'master' and toplevel == 'Settings':
```

Comment:
Although this is good as a quickfix, I agree with Pete - this should be handled elsewhere, not in `cfme.web_ui.menu`.